error_tracker/django/utils.py

```diff
@@ -182,3 +183,34 @@ def capture_exception(request=None, exception=None, additional_context=None):
     from error_tracker.django.middleware import error_tracker
     error_tracker.capture_exception(request=request, exception=exception,
                                     additional_context=additional_context)
+
+
+def clean_value(x):
+    x = x.value.replace('[["', "").replace('"]]', "").replace('"', "")
+    return x
+
+
+def cookie_parse(headers):
+    """
+    Parse request headers to extract cookie.
+    :param headers (request headers])
+    :return: [dict]: return parse header with cookie as dict
+    """
+    new_headers = {}
+    for key, value in headers.items():
+        try:
+            # Test if value could be json loaded, parse if needed as for cookie.
+            json.loads('{"%s":"%s"}' % (key, value))
```

Comment:
No usage of ` json.loads`?

Reply:
Emulate json.loads to detect problematic values and set them to ```value=""```

Reply:
Can you please provide some more details how it helps you and on what value? As you're encoding value in string something like this

```
{
   "foo" : "<value>"
}
```

As your code, any value could be parsed.